web/app/themes/dp-timber-theme/index.php

```diff
@@ -16,7 +16,8 @@
 $context          = Timber::context();
 $context['posts'] = Timber::get_posts();
 $context['foo'] = 'Shayaan';
-$templates        = array( 'index.twig' );
+$context['qux'] = 'lorem ipsum doller si amut  ';
+$templates        = array( 'variables.twig' );
```

Comment:
add partials and include it inside the index.twig

Reply:
Yes I created a basic.twig file in Partials and included it in Index.twig using 

`{% include "partial/basic.twig" %}`

Reply:
I've updated the code, please review it